exercises/luhn/test/luhn_tests.erl

```diff
@@ -6,13 +6,21 @@
 
 
 invalid_test() ->
-  ?assertNot(?TESTED_MODULE:valid("1111")),
-  ?assertNot(?TESTED_MODULE:valid("738")).
+  ?assertNot(?TESTED_MODULE:valid("1")),
+  ?assertNot(?TESTED_MODULE:valid("0")),
+  ?assertNot(?TESTED_MODULE:valid("055 444 286")),
+  ?assertNot(?TESTED_MODULE:valid("8273 1232 7352 0569")),
+  ?assertNot(?TESTED_MODULE:valid("055a 444 285")),
+  ?assertNot(?TESTED_MODULE:valid("055-444-285")),
+  ?assertNot(?TESTED_MODULE:valid("055£ 444$ 285")),
+  ?assertNot(?TESTED_MODULE:valid(" 0")).
 
 valid_test() ->
-  ?assert(?TESTED_MODULE:valid("8739567")),
-  ?assert(?TESTED_MODULE:valid("8763")),
-  ?assert(?TESTED_MODULE:valid("2323 2005 7766 3554")).
+  ?assert(?TESTED_MODULE:valid("059")),
+  ?assert(?TESTED_MODULE:valid("59")),
+  ?assert(?TESTED_MODULE:valid("055 444 285")),
+  ?assert(?TESTED_MODULE:valid("0000 0")),
+  ?assert(?TESTED_MODULE:valid("091")).
 
 create_test() ->
```

Comment:
`create/1` is not a property in the canonical data, therefore we should remove it from the test body. Please remove this test completely.